wrappers/python/c3py/c3py.py

```diff
@@ -782,8 +811,80 @@ def func(x, param=None):
 
         return tt
         
-    
+class FTparam(object):
+
+    def __init__(self, reg):
+        self.reg = reg
+        self.ftp = c3.ft_regress_get_ft_param(reg)
+        self.ft = c3.ft_param_get_ft(self.ftp)
+        self.dim = c3.ft_param_get_dim(self.ftp)
+        self.nparams = self.get_nparams()
+        self.mem = c3.sl_mem_manager_alloc(self.dim,1,self.nparams,c3.LINEAR_ST)
```

Comment:
This will only work with c3.LINEAR_ST -- so nonlinear parameterizations will not be allowed. Is this intended?